roles/postgresql/defaults/main.yml

```diff
@@ -75,6 +75,10 @@ postgresql_jupyterhub_user: jupyterhub
 postgresql_jupyterhub_password: jupyterhub
 postgresql_jupyterhub_database: jupyterhub
 
+postgresql_hue_user: hue
+postgresql_hue_password: hue
+postgresql_hue_database: hue
+
 postgresql_generated_users: >-
```

Comment:
Did you use the last commit for modifications? Because for ```hive_ms```, ```hive``` schema creation has been removed which is not used by ```hive``` application.